packages/story-editor/src/components/form/hierarchical/utils/index.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import groupBy from 'lodash.groupby';
```

Comment:
Probably added by your IDE. This needs to be imported from the design system now :-)

Reply:
Good catch! However, I just reverted that change - turns out our `groupBy` is different than the `groupBy` from lodash:

https://github.com/google/web-stories-wp/blob/main/packages/dashboard/src/utils/groupBy.js#L17-L29

Our `groupBy` doesn't create an object of arrays like lodash's does: https://lodash.com/docs/4.17.15#groupBy

Reply:
I'm happy to write our own util here though